app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :users
```

Comment:
There is a one-to-many relation between likes and users/posts, so this part should be `belongs_to :user` and `belongs_to :post`. Currently one like can belong to multiple posts and users. You should update this model as requested by the first reviewer